### PR TITLE
Implemented: option to view full-size image on click by creating image modal (#283) 

### PR DESCRIPTION
**Pull request by shanmukhdutt**
### Related Issue

#283 

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/hotwax/ionic-bopis#contribution-guideline)

### CODE REVIEW

File: src/components/ImageModal.vue [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<template>
+	<ion-page>
```
Comment (k2maan) [inline src/components/ImageModal.vue:2]: Fix indentation throughout

File: src/views/Completed.vue [hunk 2]
```diff
@@ -666,6 +667,16 @@ export default defineComponent({
       });
       return popover.present();
     },
+    async openImgModal(item: any){
```
Comment (k2maan) [inline src/views/Completed.vue:670]:
```suggestion
    async openImageModal(item: any){
```

File: src/views/Completed.vue [hunk 3]
```diff
@@ -666,6 +667,16 @@ export default defineComponent({
...
+      const imgModal = await modalController.create({
```
Comment (k2maan) [inline src/views/Completed.vue:671]:
```suggestion
      const imageModal = await modalController.create({
```

File: src/views/Completed.vue (hunk 2 above)
Comment (k2maan) [reply src/views/Completed.vue:670]: Fix in other files as well.

**Review by alsoK2maan — CHANGES_REQUESTED** [review]

File: src/components/ImageModal.vue [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  IonContent,
+  IonHeader,
+  IonIcon,
+  IonPage,
+  IonTitle,
+  IonToolbar,
+  modalController
+} from '@ionic/vue';
+export default defineComponent({
+  name: 'DisplayImg',
```
Comment (ymaheshwari1) [inline src/components/ImageModal.vue:34]:
```suggestion
  name: 'ImageModal',
```

**Review by ymaheshwari1 — COMMENTED** [review]
Update the PR with main branch and add the full size image support for kit products as well on all the pages

**Pull request closed without merging**